dataprofiler/profilers/profile_builder.py

```diff
@@ -834,6 +859,29 @@ def report(self, report_options: dict = None) -> dict:
         """
         raise NotImplementedError()
 
+    @classmethod
+    def load_from_dict(cls, data) -> BaseProfiler:
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+
+        :return: Profiler with attributes populated.
+        :rtype: BaseCompiler
+        """
+        profiler = cls(None)
```

Comment:
is `None` correct? just want to validate.

Reply:
In here might be where we need to peak into `data` and pull out labelers to set options to push down with the rest of the code when we load.

Reply:
`None` actually makes sense now that I think about it bc it avoids loading everything, doesn't it?